tezos/messages/src/p2p/encoding/limits.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) SimpleStaking and Tezedge Contributors
+// SPDX-License-Identifier: MIT
+
+/// P2P Point length
+/// `ip4:port` or `[ip6]:port`, ip4 max is 3 * 4 + 3, port max is 5
+/// ipv6 is 8 * 4 (octets) + 8 (: separators) + 2 ([]) + 5 (port)
+pub const P2P_POINT_MAX_LENGTH: usize = 8 * 4 + 8 + 2 + 5;
+
+/// Peer fixed length
+pub const PEER_ID_LENGTH: usize = 16;
+
+/// Number of validation passes/operation groups
+pub const MAX_PASS: usize = 8;
+
+/// Advertise list, max 100 items
+pub const ADVERTISE_ID_LIST_MAX_LENGTH: usize = 100;
+
+/// CurrentBranch history max length
+pub const CURRENT_BRANCH_HISTORY_MAX_LENGTH: usize = 200;
+
+/// Block header max length
+pub const BLOCK_HEADER_MAX_LENGTH: usize = 8 * 1024 * 1024;
```

Comment:
@akoptelov 
could you please, add to every constant where we can find it in ocaml code?
like we talked about max operations yesterday please

Reply:
Yes, was about to make that exact comment. At least the file and variable name containing the definition.

Reply:
Done.